tests/unit/client/feedback/schemas/test_questions.py

```diff
@@ -81,23 +106,23 @@ def test_label_question_errors(
             {
                 "type": "label_selection",
                 "options": [{"value": "a", "text": "a"}, {"value": "b", "text": "b"}],
-                "visible_options": 20,
+                "visible_options": None,
             },
         ),
         (
             {"name": "a", "description": "a", "required": True, "labels": {"a": "A", "b": "B"}},
             {
                 "type": "label_selection",
                 "options": [{"value": "a", "text": "A"}, {"value": "b", "text": "B"}],
-                "visible_options": 20,
+                "visible_options": None,
             },
         ),
         (
-            {"name": "a", "description": "a", "required": True, "labels": ["a", "b"], "visible_labels": 5},
+            {"name": "a", "description": "a", "required": True, "labels": ["a", "b"], "visible_labels": 3},
```

Comment:
this sample has already been tested in the previous test function, right? I would say to remove from here.

Reply:
Yes, I was planning to combine both and directly test `_LabelQuestion`, which is the base model for both `LabelQuestion` and `MultiLabelQuestion`, as those just add the `type` value and rely on `_LabelQuestion` for the rest of the stuff